clv/purchase_amount_model.py

```diff
@@ -253,7 +253,7 @@ def train_execute(self):
 
     def prediction_execute(self):
         print("number of users :", len(self.customers))
-        self.model_data['prediction_data'] = self.data[self.features + ["user_id"]]
+        self.model_data['prediction_data'] = self.data[self.features + [self.customer_indicator]]
```

Comment:
when the prediction is calculated for the purchase amount model_data data-frame of customer indicator column must be a generic form